### ISSUE TITLE
Magnesium not rendering properly

### ISSUE THREAD

**Issue by Morabert** [Someone-Else-Was-Taken/Periodic-Table-Reforged#114]
![Uploading 2021-11-18_04.50.58.png…](
![2021-11-18_04 36 20](https://user-images.githubusercontent.com/36409264/142456287-1edcafa2-26f2-465a-aa2f-4259b1214515.png)
![2021-11-18_04 51 06](https://user-images.githubusercontent.com/36409264/142456294-8719b985-c2f5-4108-bb55-dd477bb15cd3.png)
)
Interestingly, almost all of the blocks not being rendered are vanilla

Forge 36.2.5
MC 1.16.5
JDK 16.0.2

**Comment by NighthowlerStudios** [comment]
this looks like the chunk artifacting you can get if you did not disable flywheel's backend, or have distant horizons installed.

**Comment by Morabert** [comment]
> this looks like the chunk artifacting you can get if you did not disable flywheel's backend, or have distant horizons installed.

This has nothing to do with Flywheel.

**Comment by NighthowlerStudios** [comment]
what about distant horizons?  anything that has a custom rendering system in it could be messing with things here, I don't get this sort of artifacting in my own 240+ mod pack

**Comment by Morabert** [comment]
This is an issue tracker for the devs so that they can be made aware of bugs, not a public forum. 
Your input was not requested, nor is it relevant to the issue. Please desist.

**Comment by NighthowlerStudios** [comment]
> This is an issue tracker for the devs so that they can be made aware of bugs, not a public forum. Your input was not requested, nor is it relevant to the issue. Please desist.

I am very very aware of that.  and what I am saying is I have seen these issues happen with Flywheel and Distant Horizons.  those mods cause chunks to disappear if the pipeline for geometry rendering is changed, which results in visuals like your screenshot.  I have seen it particularly in optifine, but sodium changes chunk rendering too (which is a geometry system)  so I was thinking it was possible a mod changing rendering in your pack is causing this issue. hell I've even seen Valhelsia Structures mess with rendering in this way too.

**TL;DR** please provide a mod list so we know what we're dealing with, and/or try to find which specific mod is colliding with and causing this.  I know it's all "vanilla blocks" but that might be irrelevant here.  yes my questions are VERY relevant, as you haven't really provided enough technical information to find the cause of the issue.

**Comment by h4vok-git** [comment]
this also happened to me i disabled chunk multi draw in the settings, i just assumed it happened due to my very outdated gpu or the mods i was running.
